source/NVDAObjects/window/edit.py

```diff
@@ -674,9 +674,13 @@ def _getEmbeddedObjectLabel(self,embedRangeObj):
 
 	def _getTextAtRange(self,rangeObj):
 		embedRangeObj=None
-		bufText=rangeObj.text
+		try:
+			bufText = rangeObj.text
+		except COMError:
+			log.debugWarning("range.text failed", exc_info=True)

```

Comment:
Changing this to `rangeObj.text` instead of `range.text` to be more accurate

```suggestion
			log.debugWarning("rangeObj.text failed", exc_info=True)
```